src/_gettsim/transfers/erziehungsgeld.py

```diff
@@ -6,9 +6,8 @@
 
 @dates_active(start="2004-01-01", end="2008-12-31")
 def erziehungsgeld_m(
-    erziehungsgeld_kind_m_fg: int,
+    erziehungsgeld_eltern_m: int,
```

Comment:
So this would be just the amount before checking eligibility? We'll need some naming convention to make that clear eventually, but should only tackle that after implementing the hierarchical naming.

Reply:
Yes, `erziehungsgeld_eltern_m` is before eligibility checks on the parental level. Better naming would be something like `erziehungsgeld_vor_eltern_check_kind_m` and `erziehungsgeld_vor_eltern_check_eltern_m` but this would be super long and, as you said, easier to solve later.